From f7cc8c318aaf15b8e2fc513d2b5106194ad6b1c8 Mon Sep 17 00:00:00 2001
From: Karthikeyan Periyasamy <periyasa@codeaurora.org>
Date: Fri, 26 Jun 2020 15:29:08 +0530
Subject: [PATCH] ath11k: Fix pci get msi address

Reset the high 32 bit address with zero. so it will work in 64 bit
build since q6 runs in 32 bit address.

Signed-off-by: Karthikeyan Periyasamy <periyasa@codeaurora.org>
---
 drivers/net/wireless/ath/ath11k/pci.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/wireless/ath/ath11k/pci.c
+++ b/drivers/net/wireless/ath/ath11k/pci.c
@@ -462,6 +462,9 @@ int ath11k_pci_get_msi_address(struct at
 
 	pci_read_config_dword(pci_dev, pci_dev->msi_cap + PCI_MSI_ADDRESS_HI,
 			      msi_addr_hi);
+
+	*msi_addr_hi = 0;
+
 	return 0;
 }
 
